docs/release-notes/7.17.asciidoc

```diff
@@ -1,6 +1,16 @@
 [[release-notes-header-7.17.0]]
 == 7.17
 
+[discrete]
+[[release-notes-7.17.22]]
+=== 7.17.21
```

Comment:
```suggestion
=== 7.17.22
```

Reply:
Oops, I thought I fixed this. Good catch - thanks, @benironside!